### ISSUE TITLE
[BUG] MemoryTracker doesn't work when JEMALLOC is disabled

### ISSUE THREAD

**Issue by as51340** [memgraph/memgraph#971]
**Memgraph version**
2.8

**Environment**
Build from source

**Describe the bug**
utils::total_memory_tracker doesn't work correctly when JEMALLOC is disabled, the value is negative then. This can be seen by printing tracker's memory allocated before creating any tranasaction.

**To Reproduce**
1. Create a transaction
2. Put a breakpoint in the accessor's constructor.

**Expected behavior**
The tracker should always show positive memory allocation.

**Logs**
![image](https://github.com/memgraph/memgraph/assets/53269502/29f31335-a837-4ae7-98de-ca36018dae3c)
![image](https://github.com/memgraph/memgraph/assets/53269502/f83b0f7a-dfce-4d01-ab76-b89cdda6c5b8)

**Comment by gitbuda** [comment]
@andrejtonev is this maybe resolved/improved under https://github.com/memgraph/memgraph/pull/3061

UPDATE: Probably not, this is related to the case when jemalloc is disabled...

**Comment by andrejtonev** [comment]
@gitbuda nope, just focused on jemalloc case. I'll support this as well if we push the merge to v3.5
